content.md

```diff
@@ -26,17 +26,41 @@ A collection of tools and resources to enhance your gameplay!
 - [PP Rankings](https://pp.huismetbenen.nl/rankings/players/master) - Information and calculations for proposed and past pp reworks
 - [osu! World](https://osuworld.octo.moe/) - Provides provincial, state, and/or regional leaderboards
 ---
-### Beatmap Tools
+### Alternative Clients
+- [osu!lazer](https://github.com/ppy/osu) - Official rewrite of the game with a modern design, a bunch of new mods and features
+- [McOsu](https://store.steampowered.com/app/607260/McOsu/) - Alternative osu! client with various training tools and VR support
+- [Web osu!mania](https://web-osu-mania.vercel.app/) - Play osu!mania in the browser
+---
+### Training Tools
 - [osu!Trainer](https://github.com/FunOrange/osu-trainer) - A program that allows you to modify the difficulty of a beatmap very quickly and easily
 - [cosu!Trainer](https://github.com/hwsmm/cosutrainer) - osu!Trainer alternative for Linux, written in C
+---
+### Beatmap Management Tools
 - [Batch Beatmap Downloader](https://github.com/nzbasic/batch-beatmap-downloader) - Download many beatmaps at once, and is able to be sorted by tags such as year ranked, etc.
 - [Collection Manager](https://github.com/Piotrekol/CollectionManager) - Import and export collections made by yourself as well as other players
 ---
+### Beatmap Search Tools
+- [Advanced in-game/web search](https://osu.ppy.sh/wiki/en/Beatmap_search) - Wiki article on how to use the advanced search features in-game and on the website search bar
+- [osu!Collector](https://osucollector.com/) - Beatmap collections from the members of the osu! community
+- [OMDB](http://omdb.nyahh.net/descriptors/) - Beatmaps categorized by various descriptors such as gimmicks, styles, skillsets, and more
+- Beatmap mirrors - alternative download links for beatmaps:
+  - [osu.direct](https://osu.direct/browse) - Also provides a desktop client and a developer API
+  - [BeatConnect](https://beatconnect.io/) - Also provides a desktop client and a developer API
+  - [Mino](https://catboy.best/search) - Also provides a developer API
+  - [NeriNyan](https://nerinyan.moe/) - Also provides a developer API
```

Comment:
I'm not sure we should endorse mirrors especially considering they could hide binaries in their zips. Afaik some of them have also done some weird practices before.

Reply:
In my opinion, they are an important part of the ecosystem. In some cases beatmaps become unavailable on the official website and it's only possible to find them in a mirror.

Reply:
let's not forget it is straight up ignoring dmca claims and even more illegal than osu itself is

Reply:
I could not find any such claims elsewhere, plus I assume most mirrors would be outside of DMCA jurisdiction.

Reply:
people use these mirrors mostly to download maps that are DMCA'd, they might be dmca compliant themselves but there's not gonna be a single label going after them so that's pretty irrelevant here

Reply:
there's no way they are outside of DMCA jurisdiction as they are literally hosting copyrighted content

Reply:
DMCA is a US law and may not be enforced if the host is outside of US.

There are several legitimate use cases I can think of:
- making beatmaps available in countries where osu! is blocked,
- making beatmaps available when the official site is down,
- making beatmaps available without an account (relevant to players who are banned from the game),
- preserving graveyarded beatmaps that may be unavailable because mapper threw a fit and decided to delete them,
- preserving maps that may have been histrionically significant, but deemed inappropriate by the game staff.